vars/buildChain.groovy

```diff
@@ -0,0 +1,25 @@
+import groovy.json.JsonSlurper
+import org.yaml.snakeyaml.Yaml
+
+/*
+* It gets buildChain verion from composite action.yml file
+*/
+def getBuildChainVersionFromCompositeActionFile(String actionFilePath = '.ci/actions/build-chain/action.yml', String usesContainingString = 'github-action-build-chain@') {
```

Comment:
It might make sense to add a comment in the `action.yml` file to link this class and mention that must be keep aligned or similar. Wdyt?